docs/ProductSKUAttributes.md

```diff
@@ -0,0 +1,41 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
+
+# Product SKU Attributes
+
+The `ProductSKUAttributes` is a block responsible for **displaying a list of the currrent SKU variations** for the product.
+
+## Configuration
+
+1. Import the `vtex.store-components` app to your theme's dependencies in the `manifest.json`, for example:
+
+```json
+  "dependencies: {
+    "vtex.store-components": "3.x"
+  }
+```
+
+2. Add the `product-sku-attributes` block to any block below `store.product` (Product template). For example:
+
+```json
+  "store.product": {
+    "children": [
+      "flex-layout.row#product",
+    ]
+  },
+  "flex-layout.row#product": {
+    "children": [
+      "product-sku-attributes"
+    ]
+  },
+```
+
```

Comment:
Hey, Wender! I have a question about this step. I'm not sure if I understood it correctly, but the idea here is to declare the block anywhere? And then it is going to be automatically rendered in the "right" place on the UI?